### DISCUSSION TITLE
Allow an option to change volume in decibels

### DISCUSSION THREAD

**Discussion by tsbreuer** [ppy/osu#19385] [Ideas]
I know this will be an extremly weird edge case scenario but i thought it would be a nice idea to add. I'll explain my case so it makes a bit of sense:

Currently i have a Focusrite 2i2 interface setup with voicemeeter. The issue however is that while i can regulate the audio with voicemeeter, it introduces some audio latency over osu! directly outputting to the focusrite. 

The solution was to directly output to the focusrite, which while still has some audio latency, its still manageable. However, this has the side effect that for some reason, osu! is incredibly loud. Like, i'm running at 1% master volume and its still loud levels of loud.

An alternative i found arround this was to use eartrumpet (which i already had for different purposes) to lower osu!'s volume on the windows playback for the focusrite direct output specifically (Incredibly convoluted, i know).

It would be much simpler if osu! simply allowed to lower or increase master volume in negative or positive decibels, which would likely allow to manipulate sound levels with more granularity than % of whatever min/max volume is.

**Comment by peppy** [comment]
Sounds like a potential issue with your IS or drivers since it is not that loud for anyone else.

I'd suggest adjusting the game in windows volume mixer or equivalent.
